From e515966395ae594b220b5330ae198b76e02ac182 Mon Sep 17 00:00:00 2001
From: "Thang Q. Nguyen" <thangqn@amperecomputing.com>
Date: Tue, 3 Nov 2020 01:37:09 +0000
Subject: [PATCH] aspeed: mtjade: enable I2C4 device access.

This commit sets the BMC_GPIOY2_I2C4_O_EN signal to enable accessing FRU
EEPROM and TSx sensors on the I2C4 bus

Signed-off-by: Thang Q. Nguyen <thangqn@amperecomputing.com>
---
 board/aspeed/ast-g5/ast-g5.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/board/aspeed/ast-g5/ast-g5.c b/board/aspeed/ast-g5/ast-g5.c
index 94605b6bd9..53f0fcab31 100644
--- a/board/aspeed/ast-g5/ast-g5.c
+++ b/board/aspeed/ast-g5/ast-g5.c
@@ -88,6 +88,9 @@ int board_init(void)
 	writel(0xaa, AST_SCU_BASE);
 #endif
 
+	/* Enable I2C4 device access */
+	gpio_direction_output(194, 1);
+
 	return 0;
 }
 
-- 
2.25.1

